views/account/profile/index.ejs

```diff
@@ -14,33 +25,32 @@
     >
       <div class="row col-12">
         <main class="px-3">
-          <h1 class="display-4">Goon up Brah.</h1>
-          <h1>Hello World!</h1>
-          <p>This file was served with Express.</p>
-          <p>Here is your information:</p>
-          <ul>
-            <li>First Name: <%= user.first_name %></li>
-            <li>Last Name: <%= user.last_name %></li>
-            <li>Email: <%= user.email %></li>
-            <li>Password: <%= user.password %></li>
-            <li>
-              Settings:
-              <ul>
-                <% var settings = JSON.parse(user.settings);
-                Object.keys(settings).forEach( (key) => { %>
-                <li><%= key %>: <%= settings[key] %></li>
-                <% }); %>
-              </ul>
-            </li>
-          </ul>
-
-          <a
-            href="#"
-            class="btn btn-warning btn-lg"
-            role="button"
-            aria-disabled="true"
-            >Generate</a
-          >
+          <h1 class="display-4">Profile</h1>
```

Comment:
Little bit of a nitpick but I think if we're gonna use the "Profile" and "Casual" headers, then we don't need line 28

Reply:
> Lol you're good! Like Payton said, the tables look good. I like the buttons in the qr code page.
> 
> I didn't make an explicit comment but that link really should be changed to something else. Like "Edit Links" or something related to it.
> 
> If we do end up moving forward with these tables specifically, I would look into changing the background color so it doesn't contrast as much with the color theme. That's not really too big of a concern though

If the background color is changed, then all of the text colors would have to change to a different color, and I tried that earlier today and it was really hard to override the jquery css because they used !important for some dumb reason, so my solution was to just have a light grey background-color. But like I said before I can try to make it look more similar to what Payton has in his mockup.